#Service to run spcial command

int64 command  #integer identificating the command to run
float64 param1 #Special purpose parameter number 1
float64 param2 #Special purpose parameter number 2
float64 param3 #Special purpose parameter number 3
float64 param4 #Special purpose parameter number 4
float64 param5 #Special purpose parameter number 5
---
int64 ret
string msg
